src/main/java/frc/robot/Robot.java

```diff
@@ -23,7 +24,8 @@
   /**
    * This function is run when the robot is first started up and should be used
    * for any initialization code.
-   */
+   */ //CHASSIS motors
+  
   @Override
   public void robotInit() {
     m_oi = new OI();
```

Comment:
Same concept. If OI content is static, there is no need for a constructor.

Reply:
האמת היא, זה חלק מהתבנית שהם נותנים לקוד (עשו את זה אוטומתי), אז דווקא עדיף לא לשנות את זה